### PR TITLE
login/signup page design

### PR DESCRIPTION
**Pull request by majdmaihoub1**
please if you were **avaliable** merge these pull request please please please

thank you
download

### CODE REVIEW

File: login-signup.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		</div>
+	</div>
+</div>
+
+<footer>
+
+</footer>
+<script src="login.js"></script>
+<script src="./assets/fontawesome/js/all.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
```
Comment (slman-ali-dev) [inline login-signup.html:64]: Please use inner sources rather than outer sources.

File: login-signup.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
+<!doctype html>
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width , initial-scale=1.0">
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="assets/fontawesome/css/all.min.css">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```
Comment (slman-ali-dev) [inline login-signup.html:8]: Please use inner sources rather than outer sources.

**Review by slman-ali-dev — CHANGES_REQUESTED** [review]
Please change the lines I've selected change them and re-ask for pull request.

**Comment by majdmaihoub1** [comment]
WTF shit head???

________________________________

**Comment by slman-ali-dev** [comment]
I'll forward this message to the HR, so they can deal with whores like you.
Have a nice day :grinning:

**Pull request merged**